controllers/array_action/array_action_types.py

```diff
@@ -13,7 +13,7 @@ class Volume:
     source_id: str
     array_type: str
     pool: str
-    space_efficiency: str = None
+    space_efficiency_aliases: set = field(default_factory=set)
     default_space_efficiency: str = None
```

Comment:
do we still need this field?

Reply:
no, done (I left the UT cause to these with empty ad default is still a valid test..)